libs/SalesforceHybridSDK/SalesforceHybridSDK/Classes/SFHybridViewController.m

```diff
@@ -284,14 +283,10 @@ - (void)authenticateWithCompletionBlock:(SFOAuthPluginAuthSuccessBlock)completio
     } failure:^(SFOAuthInfo *authInfo, NSError *error) {
         if ([self logoutOnInvalidCredentials:error]) {
             [self log:SFLogLevelError msg:@"OAuth plugin authentication request failed. Logging out."];
-            SFSDKSalesforceAnalyticsManager *manager = [SFSDKSalesforceAnalyticsManager sharedInstanceWithUser:[SFUserAccountManager sharedInstance].currentUser];
-            SFSDKInstrumentationEvent *event = [SFSDKInstrumentationEventBuilder buildEventWithBuilderBlock:^(SFSDKInstrumentationEventBuilder *builder) {
-                builder.name = [NSString stringWithFormat:@"Server Error: %ld, Logout Cause: %@", (long)error.code, error.localizedDescription];
-                builder.page = @{ @"context" : @"Login Authentication"};
-                builder.schemaType = SchemaTypeInteraction;
-                builder.eventType = EventTypeUser;
-            } analyticsManager:manager.analyticsManager];
-            [manager.analyticsManager.storeManager storeEvent:event];
+            NSMutableDictionary *attributes = [[NSMutableDictionary alloc] init];
```

Comment:
We could. I did this because I can see the list of `attributes` we want to capture growing in the future, which might make inline creation less readable. But I'm okay either way. I can change it to inline.